### ISSUE TITLE
Update HelpShift lib to the latest version

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-Android#3346]
We're using 3.8 currently, seems like [3.10](https://developers.helpshift.com/android/release-notes/) fixed a frequent crash we have.

```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/com.helpshift.HSConversation}: java.lang.NullPointerException
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java)
       at android.app.ActivityThread.access$600(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java)
       at android.os.Handler.dispatchMessage(Handler.java)
       at android.os.Looper.loop(Looper.java)
       at android.app.ActivityThread.main(ActivityThread.java)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java)
       at dalvik.system.NativeStart.main(NativeStart.java)
Caused by java.lang.NullPointerException
       at android.os.Bundle.(Bundle.java)
       at com.helpshift.HSConversation.onCreate(HSConversation.java)
       at android.app.Activity.performCreate(Activity.java)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java)
       at android.app.ActivityThread.access$600(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java)
       at android.os.Handler.dispatchMessage(Handler.java)
       at android.os.Looper.loop(Looper.java)
       at android.app.ActivityThread.main(ActivityThread.java)
       at java.lang.reflect.Method.invokeNative(Method.java)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java)
       at dalvik.system.NativeStart.main(NativeStart.java)
```

**Comment by maxme** [comment]
Issue is not fixed by helpshift 3.12.0 (we have the same crash on wpandroid 4.7)

**Comment by maxme** [comment]
Updated to HelpShift 4.2 - there are new NPE :/ - I contacted their support.
